src/result.js

```diff
@@ -46,6 +47,24 @@ class Result {
   }
 
   /**
+   * Returns the serialized representation of the HEAD.
+   *
+   * @returns {String} serialized version of the DOM HEAD
+  */
+  getHead() {
```

Comment:
Should this (and `getBody`) be a real getter instead? I ever so _slightly_ dislike the `get***` and `set***` pattern when we can just use native getters/setters if that works the same for us...